hack/update-generated-bindata.sh

```diff
@@ -2,6 +2,7 @@
 STARTTIME=$(date +%s)
 source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
 
+export GO111MODULE=off
```

Comment:
go generate fails if it can't load modules?  If i run the current script on a go 1.13 system it works - why would this be needed?

Reply:
```console
$ go version
go version go1.14.1 linux/amd64
$ ./hack/update-generated-bindata.sh 
go: finding module for package github.com/openshift/origin/test/extended/authorization
go: finding module for package github.com/openshift/origin/test/extended/apiserver
---
[ERROR] hack/update-generated-bindata.sh:6: `go generate ./test/extended` exited with status 1.
```